src/index.js

```diff
@@ -1,17 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import './styles/reset.css';
+import './styles/index.scss';
+import App from './components/App';
 
 ReactDOM.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
-  document.getElementById('root')
+  document.getElementById('root'),
 );
```

Comment:
Please make sure the index has 2 <Link>s in the navbar which leads to the booklist and categories locations.
The navbar might be in its own component, that's ok!

 Please make sure the index has 2 <Route>s, one for the BookList component and the other for the Categories component. In order to not show a blank page when visiting the site for the first time, the BookList component should be the default page

I can see you have implemented this in App.j please move that here for now.

Reply:
Hi ArslanBisharat,

What do you mean by 2 s?

Thank you,
Arturo